2025/did-methods-wg.html

```diff
@@ -280,7 +280,7 @@ <h3>
                       <li><code>did:webvh</code> without domain binding (<a href="https://identity.foundation/didwebvh/v1.0/">draft specification</a>, <a href="https://github.com/decentralized-identity/did-methods/blob/main/method-proposals/PROPOSAL-did-webvh.md">standardization proposal</a>)</li>
                       <li><code>did:peer</code> (<a href="https://identity.foundation/peer-did-method-spec/">draft specification</a>, <a href="https://github.com/decentralized-identity/did-methods/blob/main/method-proposals/PROPOSAL-did-peer.md">standardization proposal</a>)</li>
                       <li><code>did:scid</code> (<a href="https://lf-toip.atlassian.net/wiki/spaces/HOME/pages/88572360/DID+SCID+Method+Specification">draft specification</a>, <a href="https://github.com/decentralized-identity/did-methods/blob/main/method-proposals/PROPOSAL-did-scid.md">standardization proposal</a>)</li>
-                      <li>a DID Method using a <a href="https://docs.google.com/presentation/d/1PKmycxKKww3FoAu-X19Pep--Z_vWZEbyjb19OcW0bSA/edit">cryptographic event log</a> of some kind</li>
+                      <li>a DID Method using a <a href="https://digitalbazaar.github.io/cel-spec/">cryptographic event log</a> of some kind</li>
```

Comment:
I'd feel better about linking to this if it were adopted as a CCG work item. I feel uncomfortable linking to any specification, in a W3C WG charter, that isn't under W3C IP control.